From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mike Hommey <mh+mozilla@glandium.org>
Date: Wed, 13 Dec 2023 02:39:24 +0000
Subject: [PATCH] Bug 1860630 - Upgrade zstandard to 0.22.
 r=firefox-build-system-reviewers,mach-reviewers,sergesanspaille,ahochheiden

Differential Revision: https://phabricator.services.mozilla.com/D194184
---
 build/zstandard_requirements.in  |   2 +-
 build/zstandard_requirements.txt | 103 +++++++++++++++----------------
 2 files changed, 50 insertions(+), 55 deletions(-)

diff --git a/build/zstandard_requirements.in b/build/zstandard_requirements.in
index 1007e454b5cf..168042c4bcaa 100644
--- a/build/zstandard_requirements.in
+++ b/build/zstandard_requirements.in
@@ -1,2 +1,2 @@
-zstandard==0.19.0
+zstandard==0.22.0
 
diff --git a/build/zstandard_requirements.txt b/build/zstandard_requirements.txt
index 078922ab42a9..7553ded463a3 100644
--- a/build/zstandard_requirements.txt
+++ b/build/zstandard_requirements.txt
@@ -1,59 +1,54 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.11
+# by the following command:
 #
 #    pip-compile --generate-hashes --output-file=build/zstandard_requirements.txt build/zstandard_requirements.in
 #
-zstandard==0.19.0 \
-    --hash=sha256:04c298d381a3b6274b0a8001f0da0ec7819d052ad9c3b0863fe8c7f154061f76 \
-    --hash=sha256:0fde1c56ec118940974e726c2a27e5b54e71e16c6f81d0b4722112b91d2d9009 \
-    --hash=sha256:126aa8433773efad0871f624339c7984a9c43913952f77d5abeee7f95a0c0860 \
-    --hash=sha256:1a4fb8b4ac6772e4d656103ccaf2e43e45bd16b5da324b963d58ef360d09eb73 \
-    --hash=sha256:2e4812720582d0803e84aefa2ac48ce1e1e6e200ca3ce1ae2be6d410c1d637ae \
-    --hash=sha256:2f01b27d0b453f07cbcff01405cdd007e71f5d6410eb01303a16ba19213e58e4 \
-    --hash=sha256:31d12fcd942dd8dbf52ca5f6b1bbe287f44e5d551a081a983ff3ea2082867863 \
-    --hash=sha256:3c927b6aa682c6d96225e1c797f4a5d0b9f777b327dea912b23471aaf5385376 \
-    --hash=sha256:3d5bb598963ac1f1f5b72dd006adb46ca6203e4fb7269a5b6e1f99e85b07ad38 \
-    --hash=sha256:401508efe02341ae681752a87e8ac9ef76df85ef1a238a7a21786a489d2c983d \
-    --hash=sha256:4514b19abe6dbd36d6c5d75c54faca24b1ceb3999193c5b1f4b685abeabde3d0 \
-    --hash=sha256:47dfa52bed3097c705451bafd56dac26535545a987b6759fa39da1602349d7ba \
-    --hash=sha256:4fa496d2d674c6e9cffc561639d17009d29adee84a27cf1e12d3c9be14aa8feb \
-    --hash=sha256:55a513ec67e85abd8b8b83af8813368036f03e2d29a50fc94033504918273980 \
-    --hash=sha256:55b3187e0bed004533149882ef8c24e954321f3be81f8a9ceffe35099b82a0d0 \
-    --hash=sha256:593f96718ad906e24d6534187fdade28b611f8ed06e27ba972ba48aecec45fc6 \
-    --hash=sha256:5e21032efe673b887464667d09406bab6e16d96b09ad87e80859e3a20b6745b6 \
-    --hash=sha256:60a86b7b2b1c300779167cf595e019e61afcc0e20c4838692983a921db9006ac \
-    --hash=sha256:619f9bf37cdb4c3dc9d4120d2a1003f5db9446f3618a323219f408f6a9df6725 \
-    --hash=sha256:660b91eca10ee1b44c47843894abe3e6cfd80e50c90dee3123befbf7ca486bd3 \
-    --hash=sha256:67710d220af405f5ce22712fa741d85e8b3ada7a457ea419b038469ba379837c \
-    --hash=sha256:6caed86cd47ae93915d9031dc04be5283c275e1a2af2ceff33932071f3eeff4d \
-    --hash=sha256:6d2182e648e79213b3881998b30225b3f4b1f3e681f1c1eaf4cacf19bde1040d \
-    --hash=sha256:72758c9f785831d9d744af282d54c3e0f9db34f7eae521c33798695464993da2 \
-    --hash=sha256:74c2637d12eaacb503b0b06efdf55199a11b1d7c580bd3dd9dfe84cac97ef2f6 \
-    --hash=sha256:755020d5aeb1b10bffd93d119e7709a2a7475b6ad79c8d5226cea3f76d152ce0 \
-    --hash=sha256:7ccc4727300f223184520a6064c161a90b5d0283accd72d1455bcd85ec44dd0d \
-    --hash=sha256:81ab21d03e3b0351847a86a0b298b297fde1e152752614138021d6d16a476ea6 \
-    --hash=sha256:8371217dff635cfc0220db2720fc3ce728cd47e72bb7572cca035332823dbdfc \
-    --hash=sha256:876567136b0359f6581ecd892bdb4ca03a0eead0265db73206c78cff03bcdb0f \
-    --hash=sha256:879411d04068bd489db57dcf6b82ffad3c5fb2a1fdd30817c566d8b7bedee442 \
-    --hash=sha256:898500957ae5e7f31b7271ace4e6f3625b38c0ac84e8cedde8de3a77a7fdae5e \
-    --hash=sha256:8c9ca56345b0c5574db47560603de9d05f63cce5dfeb3a456eb60f3fec737ff2 \
-    --hash=sha256:8ec2c146e10b59c376b6bc0369929647fcd95404a503a7aa0990f21c16462248 \
-    --hash=sha256:8f7c68de4f362c1b2f426395fe4e05028c56d0782b2ec3ae18a5416eaf775576 \
-    --hash=sha256:909bdd4e19ea437eb9b45d6695d722f6f0fd9d8f493e837d70f92062b9f39faf \
-    --hash=sha256:9d97c713433087ba5cee61a3e8edb54029753d45a4288ad61a176fa4718033ce \
-    --hash=sha256:a65e0119ad39e855427520f7829618f78eb2824aa05e63ff19b466080cd99210 \
-    --hash=sha256:aa9087571729c968cd853d54b3f6e9d0ec61e45cd2c31e0eb8a0d4bdbbe6da2f \
-    --hash=sha256:aef0889417eda2db000d791f9739f5cecb9ccdd45c98f82c6be531bdc67ff0f2 \
-    --hash=sha256:b253d0c53c8ee12c3e53d181fb9ef6ce2cd9c41cbca1c56a535e4fc8ec41e241 \
-    --hash=sha256:b80f6f6478f9d4ca26daee6c61584499493bf97950cfaa1a02b16bb5c2c17e70 \
-    --hash=sha256:be6329b5ba18ec5d32dc26181e0148e423347ed936dda48bf49fb243895d1566 \
-    --hash=sha256:c7560f622e3849cc8f3e999791a915addd08fafe80b47fcf3ffbda5b5151047c \
-    --hash=sha256:d1a7a716bb04b1c3c4a707e38e2dee46ac544fff931e66d7ae944f3019fc55b8 \
-    --hash=sha256:d63b04e16df8ea21dfcedbf5a60e11cbba9d835d44cb3cbff233cfd037a916d5 \
-    --hash=sha256:d777d239036815e9b3a093fa9208ad314c040c26d7246617e70e23025b60083a \
-    --hash=sha256:e892d3177380ec080550b56a7ffeab680af25575d291766bdd875147ba246a91 \
-    --hash=sha256:e9c90a44470f2999779057aeaf33461cbd8bb59d8f15e983150d10bb260e16e0 \
-    --hash=sha256:f097dda5d4f9b9b01b3c9fa2069f9c02929365f48f341feddf3d6b32510a2f93 \
-    --hash=sha256:f4ebfe03cbae821ef994b2e58e4df6a087470cc522aca502614e82a143365d45
+zstandard==0.22.0 \
+    --hash=sha256:11f0d1aab9516a497137b41e3d3ed4bbf7b2ee2abc79e5c8b010ad286d7464bd \
+    --hash=sha256:1958100b8a1cc3f27fa21071a55cb2ed32e9e5df4c3c6e661c193437f171cba2 \
+    --hash=sha256:1a90ba9a4c9c884bb876a14be2b1d216609385efb180393df40e5172e7ecf356 \
+    --hash=sha256:1d43501f5f31e22baf822720d82b5547f8a08f5386a883b32584a185675c8fbf \
+    --hash=sha256:23d2b3c2b8e7e5a6cb7922f7c27d73a9a615f0a5ab5d0e03dd533c477de23004 \
+    --hash=sha256:2612e9bb4977381184bb2463150336d0f7e014d6bb5d4a370f9a372d21916f69 \
+    --hash=sha256:275df437ab03f8c033b8a2c181e51716c32d831082d93ce48002a5227ec93019 \
+    --hash=sha256:2ac9957bc6d2403c4772c890916bf181b2653640da98f32e04b96e4d6fb3252a \
+    --hash=sha256:2b11ea433db22e720758cba584c9d661077121fcf60ab43351950ded20283440 \
+    --hash=sha256:2fdd53b806786bd6112d97c1f1e7841e5e4daa06810ab4b284026a1a0e484c0b \
+    --hash=sha256:33591d59f4956c9812f8063eff2e2c0065bc02050837f152574069f5f9f17775 \
+    --hash=sha256:36a47636c3de227cd765e25a21dc5dace00539b82ddd99ee36abae38178eff9e \
+    --hash=sha256:39b2853efc9403927f9065cc48c9980649462acbdf81cd4f0cb773af2fd734bc \
+    --hash=sha256:3db41c5e49ef73641d5111554e1d1d3af106410a6c1fb52cf68912ba7a343a0d \
+    --hash=sha256:445b47bc32de69d990ad0f34da0e20f535914623d1e506e74d6bc5c9dc40bb09 \
+    --hash=sha256:466e6ad8caefb589ed281c076deb6f0cd330e8bc13c5035854ffb9c2014b118c \
+    --hash=sha256:48f260e4c7294ef275744210a4010f116048e0c95857befb7462e033f09442fe \
+    --hash=sha256:4ac59d5d6910b220141c1737b79d4a5aa9e57466e7469a012ed42ce2d3995e88 \
+    --hash=sha256:53866a9d8ab363271c9e80c7c2e9441814961d47f88c9bc3b248142c32141d94 \
+    --hash=sha256:589402548251056878d2e7c8859286eb91bd841af117dbe4ab000e6450987e08 \
+    --hash=sha256:68953dc84b244b053c0d5f137a21ae8287ecf51b20872eccf8eaac0302d3e3b0 \
+    --hash=sha256:6c25b8eb733d4e741246151d895dd0308137532737f337411160ff69ca24f93a \
+    --hash=sha256:7034d381789f45576ec3f1fa0e15d741828146439228dc3f7c59856c5bcd3292 \
+    --hash=sha256:73a1d6bd01961e9fd447162e137ed949c01bdb830dfca487c4a14e9742dccc93 \
+    --hash=sha256:8226a33c542bcb54cd6bd0a366067b610b41713b64c9abec1bc4533d69f51e70 \
+    --hash=sha256:888196c9c8893a1e8ff5e89b8f894e7f4f0e64a5af4d8f3c410f0319128bb2f8 \
+    --hash=sha256:88c5b4b47a8a138338a07fc94e2ba3b1535f69247670abfe422de4e0b344aae2 \
+    --hash=sha256:8a1b2effa96a5f019e72874969394edd393e2fbd6414a8208fea363a22803b45 \
+    --hash=sha256:93e1856c8313bc688d5df069e106a4bc962eef3d13372020cc6e3ebf5e045202 \
+    --hash=sha256:9501f36fac6b875c124243a379267d879262480bf85b1dbda61f5ad4d01b75a3 \
+    --hash=sha256:959665072bd60f45c5b6b5d711f15bdefc9849dd5da9fb6c873e35f5d34d8cfb \
+    --hash=sha256:a1d67d0d53d2a138f9e29d8acdabe11310c185e36f0a848efa104d4e40b808e4 \
+    --hash=sha256:a493d470183ee620a3df1e6e55b3e4de8143c0ba1b16f3ded83208ea8ddfd91d \
+    --hash=sha256:a7ccf5825fd71d4542c8ab28d4d482aace885f5ebe4b40faaa290eed8e095a4c \
+    --hash=sha256:a88b7df61a292603e7cd662d92565d915796b094ffb3d206579aaebac6b85d5f \
+    --hash=sha256:a97079b955b00b732c6f280d5023e0eefe359045e8b83b08cf0333af9ec78f26 \
+    --hash=sha256:d22fdef58976457c65e2796e6730a3ea4a254f3ba83777ecfc8592ff8d77d303 \
+    --hash=sha256:d75f693bb4e92c335e0645e8845e553cd09dc91616412d1d4650da835b5449df \
+    --hash=sha256:d8593f8464fb64d58e8cb0b905b272d40184eac9a18d83cf8c10749c3eafcd7e \
+    --hash=sha256:d8fff0f0c1d8bc5d866762ae95bd99d53282337af1be9dc0d88506b340e74b73 \
+    --hash=sha256:de20a212ef3d00d609d0b22eb7cc798d5a69035e81839f549b538eff4105d01c \
+    --hash=sha256:e9e9d4e2e336c529d4c435baad846a181e39a982f823f7e4495ec0b0ec8538d2 \
+    --hash=sha256:f058a77ef0ece4e210bb0450e68408d4223f728b109764676e1a13537d056bb0 \
+    --hash=sha256:f1a4b358947a65b94e2501ce3e078bbc929b039ede4679ddb0460829b12f7375 \
+    --hash=sha256:f9b2cde1cd1b2a10246dbc143ba49d942d14fb3d2b4bccf4618d475c65464912 \
+    --hash=sha256:fe3390c538f12437b859d815040763abc728955a52ca6ff9c5d4ac707c4ad98e
     # via -r build/zstandard_requirements.in
